PyDSS/pyPostprocessor/PostprocessScripts/AutomatedVoltageUpgrade.py

```diff
@@ -1945,17 +1935,16 @@ def apply_best_LTCsetting(self, upper_limit):
         #  by upper voltage limit - basically the maximum possible severity
         min_severity = 10000000000000
         # min_severity = pow(len(self.all_bus_names), 2) * len(self.config["tps_to_test"]) * upper_limit
-        print("Severity: ", pow(len(self.all_bus_names), 2) * len(self.config["tps_to_test"]) * upper_limit)
-        print(self.subLTC_sweep_viols)
+        self.logger.info(f"Severity: {pow(len(self.all_bus_names), 2) * len(self.config['tps_to_test']) * upper_limit}")
+        self.logger.debug(self.subLTC_sweep_viols)
         for key, val in self.subLTC_sweep_viols.items():
             if val < min_severity:
                 min_severity = val
                 best_setting = key
         if best_setting == "original":
             self.apply_orig_LTC_setting()
         else:
-            print("\n\nbest_setting")
-            print(best_setting)
+            self.logger.debug(f"\n\nBest_setting: {best_setting}")

```

Comment:
In all logging statements you should use the `%s` formatter instead of string interpolation.  The reason is that the string interpolation will be executed even if the log level is disabled.  This is particularly important for debug statements that will be executed many times. That may not be the case here; this is just a general point.

Also, no need for the `\n`.

```suggestion
            self.logger.debug("Best_setting: %s", best_setting")

```